addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -877,10 +873,10 @@ odoo.define('website.tour.form_editor', function (require) {
             run: () => null,
         }, {
             content: 'Check the first checkbox',
-            trigger: 'input[type="checkbox"][name="Checkbox 1"]',
+            trigger: `input[type="checkbox"][name="${getEncodedName('Checkbox 1')}"]`,
```

Comment:
Mmh, annoying. Maybe `encodeURIComponent` is not the way to go :shrug: Space was not issue in the end, and did not need to be encoded... no other way?